extensions/hibernate-search-orm-elasticsearch/deployment/src/test/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+quarkus.datasource.db-kind=h2
+quarkus.datasource.jdbc.url=jdbc:h2:mem:default;DB_CLOSE_DELAY=-1
+
+quarkus.hibernate-orm.dialect=org.hibernate.dialect.H2Dialect
+quarkus.hibernate-orm.database.generation=drop-and-create
+
+quarkus.hibernate-search-orm.elasticsearch.version=7
+quarkus.hibernate-search-orm.elasticsearch.hosts=${elasticsearch.hosts:localhost:9200}
+quarkus.hibernate-search-orm.elasticsearch.protocol=${elasticsearch.protocol:http}
+quarkus.hibernate-search-orm.schema-management.strategy=drop-and-create-and-drop
+quarkus.hibernate-search-orm.automatic-indexing.synchronization.strategy=sync
```

Comment:
Don't you expect the configuration to be potentially different depending on the PU?

Doesn't need to be part of this PR, just asking.

Reply:
Right now you can make do by defining one backend per PU, but some settings (like this one) will have to be the same on all PUs, which is not great.

So yes, we'd need additional changes to fully support multiple PUs. In another PR :)